_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -100,10 +100,18 @@ This `input` element is not [focusable][] because of the `disabled` attribute.
 This `a` element is not [focusable][] because it moves focus to the `input` element whenever it receives focus.
 
 ```html
+<div role="dialog" aria-label="Sample Modal" aria-modal="true" style="border: solid black 1px; padding: 1rem;">
+    <label>Some input <input id="dialogFirst"></label><br />
+    <button>Close button</button>
```

Comment:
Should this be a functional button? Otherwise we end up with half a keyboard trap on this page.

Reply:
Added script so that the close makes the dialog display:none